### PR TITLE
Build fixes

### PR DESCRIPTION
**Pull request by pascalberger**
Fix build

### CODE REVIEW

File: setup.cake [hunk 1]
```diff
@@ -11,7 +11,7 @@ BuildParameters.SetParameters(
     repositoryName: "Cake.Yeoman",
     appVeyorAccountName: "cakecontrib");
 
-BuildParameters.PrintParamters(Context);
+BuildParameters.PrintParameters(Context);
```
Comment (gep13) [inline setup.cake]: @pascalberger aw man, I forgot this was in each setup.cake file.  I'll have to remember to put this into each repo.

File: tools/packages.config [hunk 2]
```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
 <packages>
-	<package id="Cake" version="0.18.0-alpha0123" />
+	<package id="Cake" version="0.18.0-alpha0033" />
```
Comment (gep13) [inline tools/packages.config]: @daveaglick do you know, roughly, when you are planning on shipping the latest Cake.Wyam addin?  It is the last one that is being used by Cake.Recipe that hasn't been updated to use 0.17.0 yet.  @pascalberger sent a PR through to update it recently.  Just wondering when we can expect that to ship.  Thanks!

File: setup.cake (hunk 1 above)
Comment (pascalberger) [inline setup.cake]: @gep13 Btw is there a reason why this is in each `setup.cake` file and not in the `build.cake` file inside Cake.Recipe?

Comment (gep13) [inline setup.cake]: @pascalberger hmmm, no, I guess not. Don't remember why I opted to put it here.

**Pull request merged**